src/java.desktop/macosx/native/libawt_lwawt/awt/AWTSurfaceLayers.m

```diff
@@ -68,10 +69,11 @@ - (void) setLayer:(CALayer *)newLayer {
     }
 }
 
-// Updates back buffer size of the layer if it's an OpenGL layer
-// including all OpenGL sublayers
+// Updates back buffer size of the layer if it's an OpenGL/Metal layer
```

Comment:
Did we test that jawt functionality works in the case of metal?

Reply:
No

Reply:
@mrserb, please let us know if there are existing jawt test case(s)?

Reply:
Will be handled as JDK-8261140